src/components/calcite-pagination/calcite-pagination.scss

```diff
@@ -1,99 +1,97 @@
 // explicit px values until we add a spacing unit and new font scale to base
 :host([scale="s"]) {
-  --calcite-pagination-spacing: 4px 8px;
+  --calcite-pagination-spacing: theme("spacing.1") theme("spacing.2");
   & .previous,
   & .next,
   & .page {
-    font-size: 12px;
+    @apply text--2h;
   }
 }
 
 :host([scale="m"]) {
-  --calcite-pagination-spacing: 8px 12px;
+  --calcite-pagination-spacing: theme("spacing.2") theme("spacing.3");
   & .previous,
   & .next,
   & .page {
-    font-size: 16px;
+    @apply text-0h;
   }
 }
 
 :host([scale="l"]) {
-  --calcite-pagination-spacing: 12px 16px;
+  --calcite-pagination-spacing: theme("spacing.3") theme("spacing.4");
   & .previous,
   & .next,
   & .page {
-    font-size: 20px;
+    @apply text-2h;
   }
 }
 
 :host {
-  display: inline-flex;
+  @apply flex;
   writing-mode: horizontal-tb;
 }
 
 // focus styles
 :host button {
-  @include focus-style-base();
+  @apply focus-base;
   &:focus {
-    @include focus-style-inset();
+    @apply focus-inset;
   }
 }
 
 .previous,
 .next,
 .page {
-  display: flex;
-  align-items: center;
-  background-color: transparent;
-  border: none;
-  border-top: 3px solid transparent;
-  border-bottom: 3px solid transparent;
-  font-family: inherit;
-  @include font-size(0);
-  color: var(--calcite-ui-text-3);
-  cursor: pointer;
+  @apply flex
+    items-center
+    bg-transparent
+    border-none
+    border-opacity-0
+    text-0h
+    text-color-3
+    cursor-pointer
+    font-inherit;
+  border-top: 2px solid transparent;
+  border-bottom: 2px solid transparent;
   &:hover {
-    color: var(--calcite-ui-text-1);
+    @apply text-color-1;
     transition: all $transition;
   }
 }
 .page {
   &:hover {
-    border-bottom-color: var(--calcite-ui-border-2);
+    @apply border-b-color-2;
   }
   &.is-selected {
-    font-weight: 500;
-    color: var(--calcite-ui-text-1);
-    border-bottom-color: var(--calcite-ui-brand);
+    @apply font-medium text-color-1 border-b-color-brand;
   }
 }
 .previous,
 .next {
   padding: var(--calcite-pagination-spacing);
   &:hover {
+    @apply bg-foreground-2;
+    //FIXME: Does we have a brand color for text?
     color: var(--calcite-ui-brand);
```

Comment:
we could use the `link` color here. Probably brand is an accessibility issue when pagination is used on the light gray bg...

Reply:
@bstifle, ^ this okay?